### DISCUSSION TITLE
How does the base role work?

### DISCUSSION THREAD

**Discussion by nekrich** [community/community#126697] [Enterprise] [answered]
### Body

Hi.

My goal is to give Write access to Internal repositories for all org members.
I have a group with all org members listed, but adding it to every repo looks like overhead (even utilizing API).

So the question is about the [Base permissions](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/setting-base-permissions-for-an-organization#setting-base-permissions).
I want to clarify this message from the documentation:
> You can set base permissions that apply to all members of an organization when accessing **any of the organization's repositories**.

Does it work for **ALL** repositories, even private ones, or is it just for Internal ones?

Thanks.

**Comment by queenofcorgis** [comment]
Hi there @nekrich, 

Thanks for asking this in the Community. Is this a matter of controlling who can merge or repository visibility?

**Reply by nekrich** [reply, marked as answer]
OMG, just found the [Improved innersource collaboration and enterprise fork policies
](https://github.blog/changelog/2022-06-27-improved-innersource-collaboration-and-enterprise-fork-policies/) article, and I think it covers all our needs.

Thanks.

**Reply by queenofcorgis** [reply]
That is so great to hear - thanks for linking and marking the answer @nekrich ❤️
